src/basic/terminal-util.c

```diff
@@ -2427,6 +2445,158 @@ int terminal_get_size_by_dsr(
         return r;
 }
 
+#define MAX_TERMINFO_LENGTH 64
+/* python -c 'print("".join(hex(ord(i))[2:] for i in "TN").upper())' */
+#define DCS_TERMINFO_Q ANSI_DCS "+q" "544E" ANSI_ST
```

Comment:
As discussed, correct would be `name` (`6e616d65` in hex) here instead of the `TN`:

> - TN for termcap name (or **_name_ for terminfo name**).

https://invisible-island.net/xterm/ctlseqs/ctlseqs.html#h4-Device-Control-functions:DCS-plus-q-Pt-ST.F95

Reply:
`TN` -> `name`